src/clients/redis.ts

```diff
@@ -0,0 +1,50 @@
+import redis from 'redis';
+import { configuration } from '../config';
+import { LoggerService } from '../logger.service';
+
+export class RedisService {
+  client: redis.RedisClient;
+
+  constructor() {
+    this.client = redis.createClient({
+      db: configuration.redis.db,
+      host: configuration.redis.host,
+      port: configuration.redis.port,
+      auth_pass: configuration.redis.auth,
+    });
+
+    if (this.client.connected) {
+      LoggerService.log('✅ Redis connection is ready');
+    } else {
+      LoggerService.error('❌ Redis connection is not ready');
+      throw new Error('Redis connection error');
+    }
+  }
+
+  getJson(key: string): void {
+    this.client.get(key, (err, reply) => {
+      if (err) {
+        LoggerService.error('Error while getting key from redis with message:', err, 'RedisService');
+      }
+      return reply;
```

Comment:
this return should never get hit, as it's in a callback method (asynchronous), so this "getJson" method will always return void... I think we discussed this already? Please update this service to reflect.

Reply:
Yes - I updated that file before the conversation. Fixed 👍